frontend/src/scenes/data-warehouse/DataWarehouseScene.tsx

```diff
@@ -1,44 +1,291 @@
-import { useValues } from 'kea'
+import { useActions, useValues } from 'kea'
+
+import { IconCheckCircle, IconInfo, IconPlusSmall } from '@posthog/icons'
+import { LemonButton, LemonCard, LemonTag, Tooltip } from '@posthog/lemon-ui'
 
 import { NotFound } from 'lib/components/NotFound'
+import { TZLabel } from 'lib/components/TZLabel'
 import { FEATURE_FLAGS } from 'lib/constants'
+import { LemonTable, LemonTableColumns } from 'lib/lemon-ui/LemonTable'
+import { LemonTableLink } from 'lib/lemon-ui/LemonTable/LemonTableLink'
+import { PaginationControl, usePagination } from 'lib/lemon-ui/PaginationControl'
+import { IconCancel, IconExclamation, IconRadioButtonUnchecked, IconSync } from 'lib/lemon-ui/icons'
 import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
 import { SceneExport } from 'scenes/sceneTypes'
+import { urls } from 'scenes/urls'
 
-export const scene: SceneExport = {
-    component: DataWarehouseScene,
-}
+import { DataWarehouseActivityRecord, DataWarehouseDashboardDataSource, PipelineTab } from '~/types'
+
+import { dataWarehouseSceneLogic } from './dataWarehouseSceneLogic'
+
+export const scene: SceneExport = { component: DataWarehouseScene }
```

Comment:
you can add the dataWarehouseSceneLogic here so that it mounts alongside the scene